src/features/reports/readme.md

```diff
@@ -0,0 +1,86 @@
+## Reports
+
+This feature is responsible for presenting a list of Kontur reports and browsing them.
+Each report has a link, a description, date of latest update and report's content represented as a table. Originally those reports are stored as `.csv` tables
+
+### How to use
+
+There are 2 separate root components for that feature.
+`<ReportsList />` shows the list of reports, their brief description and general description of reports section. It also allows to navigate to the report by clicking on its card.
+`<ReportInfo />` provides whole info about the given report, taking it's id from the link.
```

Comment:
It's not clear where the `.csv` data comes from, does it fetches when the component mounts by report id?